tests/test_mgxs_library_mesh/results_true.dat

```diff
@@ -42,10 +42,10 @@
 3      2  2  1        1   total  0.021015  0.004911
   mesh 1       group in nuclide          mean      std. dev.
        x  y  z                                              
-0      1  1  1        1   total  1.596881e+06  348355.002874
-1      1  2  1        1   total  1.610266e+06  638514.779022
-2      2  1  1        1   total  2.048208e+06  257681.650350
-3      2  2  1        1   total  1.660283e+06  390011.381149
+0      1  1  1        1   total  1.596882e+06  348355.009577
+1      1  2  1        1   total  1.610266e+06  638514.780884
+2      2  1  1        1   total  2.048208e+06  257681.651022
+3      2  2  1        1   total  1.660283e+06  390011.380879
```

Comment:
When I update this test, these results revert back to what they were before. Would you expect this particular set of results to change as a result of your changes?